distribution: Fix sporadic here-script input failure (#1809)
os-autoinst/os-autoinst: doc/backend_vars.asciidoc

PAUSE_ON_SCREEN_MISMATCH;boolean;0;Pause test execution on the next screen mismatch. Same notes as for `PAUSE_AT` apply.
PAUSE_ON_NEXT_COMMAND;boolean;0;Pause test execution on the next test API command. Same notes as for `PAUSE_AT` apply.
_QUIET_SCRIPT_CALLS;boolean;0;Add quiet flag to all the calls to script_run, script_output and validate_script_output. It will omit all the squares "wait_serial expected" on the Details view of the test case. This option might be useful for serial terminal tests.

> _WAIT_STILL_SCREEN_ON_HERE_DOC_INPUT;float;0;If this value is greater then 0, we will use the amount of seconds for wait_still_screen before start typing into the here document in "script_output()" function (see: poo#60566). By default this depends on the backend.

Author: changed to 
```suggestion
_WAIT_STILL_SCREEN_ON_HERE_DOC_INPUT;float;0;If this value is greater then 0, it is used by `wait_still_screen` before starting to write the script into the here document used in `testapi::script_output()` function (see: poo#60566). By default this depends on the backend.
```

> _WAIT_STILL_SCREEN_ON_HERE_DOC_INPUT;float;0;If this value is greater then 0, we will use the amount of seconds for wait_still_screen before start typing into the here document in "script_output()" function (see: poo#60566). By default this depends on the backend.

Reviewer: > 

If the initial proposal is not good enough, I would prefer to go for this one as it is crystal clear what is the effect of setting the variable :)
